### PR TITLE
Update zh_tw.json

### PR DESCRIPTION
**Pull request by HJ-zhtw**
1. Updated added sentences for 1.14.
2. Better (maybe) localized for Taiwanese.
    A. Removed variants that Taiwanese don't use nowadays.
    B. Rewritten some vocabularies that aren't localized for Taiwanese.
    C. Added blanks between English and Chinese.
3. Rewritten some sentences.

P.S. If anything wrong, please just fix it or you could contact me with Telegram: HJ_25.

### CODE REVIEW

**Comment by mezz** [comment]
Thank you!
@hugoalh can you review these changes and let me know if they look good to you?

File: src/main/resources/assets/jei/lang/zh_tw.json [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+  "jei.tooltip.not.enough.space": "這裡沒有足夠的空間來顯示 JEI 物品列表",
+  "jei.tooltip.ingredient.list.disabled": "JEI 物品列表疊層已停用",
+  "jei.tooltip.ingredient.list.disabled.how.to.fix": "按下 %s 以啟用",
   "jei.tooltip.bookmarks": "JEI書籤",
-  "jei.tooltip.bookmarks.usage.nokey": "在按鍵設置中為JEI書籤添加綁定鍵",
-  "jei.tooltip.bookmarks.usage.key": "將鼠標懸停在某個物品上，然後按下%s以將其添加為書籤",
-  "jei.tooltip.bookmarks.not.enough.space": "這裏沒有足夠的空間來顯示JEI書籤",
+  "jei.tooltip.bookmarks.usage.nokey": "在按鍵設定中為 JEI 書籤新增綁定鍵",
+  "jei.tooltip.bookmarks.usage.key": "將游標停在某個物品上，然後按下 %s 以新增至書籤",
+  "jei.tooltip.bookmarks.not.enough.space": "這裡沒有足夠的空間來顯示JEI書籤",
```
Comment (hugoalh) [inline src/main/resources/assets/jei/lang/zh_tw.json]: > 2C. Added blanks between English and Chinese.

Forgot to spacing.

Comment (HJ-zhtw) [reply src/main/resources/assets/jei/lang/zh_tw.json]: Added.

File: src/main/resources/assets/jei/lang/zh_tw.json [hunk 2]
```diff
@@ -43,81 +43,85 @@
...
+  "config.jei.interface.bookmarkOverlayEnabled.comment": "在打開的 GUI 一側顯示書籤物品列表",
   "config.jei.search": "搜尋選項",
   "config.jei.search.comment": "搜尋欄的相關選項",
   "config.jei.search.modNameSearchMode": "@ModName",
   "config.jei.search.modNameSearchMode.comment": "依模組名稱搜尋（前綴：@）",
   "config.jei.search.tooltipSearchMode": "#Tooltip",
   "config.jei.search.tooltipSearchMode.comment": "依物品提示搜尋（前綴：#）",
   "config.jei.search.tagSearchMode": "$Tag",
-  "config.jei.search.tagSearchMode.comment": "Search mode for Tag Names (prefix: $)",
+  "config.jei.search.tagSearchMode.comment": "Search mode for Tag Names (前綴：$)",
```
Comment (hugoalh) [inline src/main/resources/assets/jei/lang/zh_tw.json]: Can translate to `依標籤搜尋（前綴：$)`

Comment (HJ-zhtw) [reply src/main/resources/assets/jei/lang/zh_tw.json]: Forgot to say, due to an unknown issue (maybe a bug), I couldn't open the Config UI. I chose not to translate some sentences that I didn't see exactly.

File: src/main/resources/assets/jei/lang/zh_tw.json [hunk 3]
```diff
@@ -43,81 +43,85 @@
...
+  "gui.jei.category.smelting.experience": "%s 經驗",
   "gui.jei.category.fuel": "燃料",
-  "gui.jei.category.fuel.smeltCount.single": "燒煉1個物品",
-  "gui.jei.category.fuel.smeltCount": "燒煉%s個物品",
+  "gui.jei.category.fuel.smeltCount.single": "可熔煉 1 個物品",
+  "gui.jei.category.fuel.smeltCount": "可熔煉 %s 個物品",
   "gui.jei.category.brewing": "釀造",
-  "gui.jei.category.brewing.steps": "需要步驟：%s",
-  "gui.jei.category.itemInformation": "描述",
+  "gui.jei.category.brewing.steps": "步驟： %s",
```
Comment (hugoalh) [inline src/main/resources/assets/jei/lang/zh_tw.json]: Is it necessary to have one whitespace after `：`?

Comment (HJ-zhtw) [reply src/main/resources/assets/jei/lang/zh_tw.json]: That was a mistake. Will be removed.

**Review by hugoalh — APPROVED** [review]

**Comment by hugoalh** [comment]
@mezz Mark as approve, look okay. 👌

**Comment by mezz** [comment]
Thank you both!

**Pull request merged**